Source/JavaScriptCore/runtime/CachedTypes.cpp

```diff
@@ -440,11 +440,6 @@ class VariableLengthObject : public CachedObject<Source>, VariableLengthObjectBa
     }
 
     template<typename T>
-#if CPU(ARM64) && CPU(ADDRESS32)
-    // FIXME: Remove this once it's no longer needed and LLVM doesn't miscompile us:
-    // <rdar://problem/49792205>
```

Comment:
It looks like this was fixed back in 2019, so should be safe to remove.

Reply:
Is it actually fixed? I think it is closed without much confirmation, so we do not know.

Reply:
Oh, I see. The radar was closed under the belief that upstream LLVM changes taken around the time fixed it, but no definitive commit for the fix.

If you are uncomfortable changing this bit, we should just keep it.

Reply:
See the list of commits that are relevant to llvm clang that have been done over the years.

Reply:
I'm unconfortable with this removal unless (1) the author ensured that this is fixed by testing or (2) the author points out the exact fix in LLVM and explain why this is already fixed. Just saying "there are many commits so it is probably already fixed" is not acceptable.

Reply:
https://github.com/llvm/llvm-project/commit/f1c28929125400a1680868f7c6eea720de256779